packages/block-editor/src/index.js

```diff
@@ -17,3 +17,4 @@ export * from './elements';
 export * from './utils';
 export { storeConfig, store } from './store';
 export { SETTINGS_DEFAULTS } from './store/defaults';
+export { parse as __internalCssParser } from './utils/transform-styles/ast'
```

Comment:
```suggestion
export { parse as __unstableCssParser } from './utils/transform-styles/ast';
```

I've not seen `__internal` used in a code base, but no strong opinion here. Also, see the guidelines - https://developer.wordpress.org/block-editor/contributors/code/coding-guidelines/#experimental-and-unstable-apis.

Reply:
Yeah, I am very torn on what a good name for this export would be. The goal here is for this to never get stabilized into an actual API that people can rely on. So maybe `__unstable` is a good fit. :) 

I'll apply your suggestion but will leave this comment open for further thoughts from any other reviewers.